Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/TextToolbar.cs

```diff
@@ -36,9 +36,9 @@ public partial class TextToolbar : Control
         internal const string HeadersElement = "Headers";
 
         /// <summary>
-        /// Gets access to Generic Buttons that activate Formatter Methods
+        /// Gets access to Generic Buttons to instantiate for Creating Formatters (does not get the Current Instance from Toolbar)
         /// </summary>
-        internal CommonButtons CommonButtons { get; }
+        public CommonButtons CommonButtons { get; }
```

Comment:
As in If you want to use the Common Buttons, you instantiate it yourself inside the formatter?
That could make sense.